### ISSUE TITLE
@const raises a "no-undef" false positive

### ISSUE THREAD

**Issue by kevin-legion** [sveltejs/eslint-plugin-svelte3#172]
Using the "no-undef" rule makes ESlint raise a "no-undef" error:

![false-positive](https://user-images.githubusercontent.com/94377405/155378266-bc48704c-2a21-4f26-bfdd-0001baed4ac1.PNG)

eslint configuration:

```js
module.exports = {
  env: {
    browser: true,
    es2021: true,
    node: true,
  },
  parser: "@typescript-eslint/parser",
  plugins: ["svelte3", "@typescript-eslint"],
  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
  overrides: [
    {
      files: ["*.svelte"],
      processor: "svelte3/svelte3",
    },
  ],
  settings: {
    "svelte3/typescript": true,
    "svelte3/ignore-styles": () => true,
  },
  rules: {
    camelcase: "error",
    "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
    "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off",
    "@typescript-eslint/no-unused-vars": "off",
    "no-var": "error",
    "padding-line-between-statements": [
      "error",
      { blankLine: "always", prev: "*", next: "return" },
      { blankLine: "always", prev: "*", next: "block-like" },
      { blankLine: "always", prev: ["const", "let", "var"], next: "*" },
      {
        blankLine: "any",
        prev: ["const", "let", "var"],
        next: ["const", "let", "var"],
      },
    ],
    "prefer-const": [
      "error",
      {
        destructuring: "any",
        ignoreReadBeforeAssign: false,
      },
    ],
  },
};
```

Svelte 3.46.4
eslint 8.9.0
eslint-plugin-svelte 3.4.0

**Comment by ota-meshi** [comment]
Hi @kevin-legion.
It seems that the fix was released 2 hours before you posted the issue.
https://github.com/sveltejs/eslint-plugin-svelte3/blob/master/CHANGELOG.md#341

**Comment by Conduitry** [comment]
Yup, duplicate of #164, already fixed in 3.4.1.

**Comment by kevin-legion** [comment]
Thank you for the quick reply, I'll update to the last version 👍
